foladipo/docs-cabinet-cp2
# Feature/148194879/admin can see users list

#### What does this PR do?

Enables admin users to see a list of all the users of this app. It also adds an 'admins only' section to the side menu.

#### Description of task to be completed

Enable admin users to see a list of all users.

#### How should this be manually tested?

- Follow the instructions stated in this project's README to set up your local instance of this app.

- Consequently, you should have set up an admin account. Log in to the app using the credentials you set in that step.

- In the app menu, you will see an option to tagged 'Users'. Click on it to see a list of all users.

- Log out of the app and create/login with a regular user account. Check the app menu: the 'Users' option should be absent.

#### Any background context you want to provide?

N/A

#### What are the relevant Pivotal Tracker stories?

148194879

#### Screenshots
<img width="1280" alt="screen shot 2017-07-03 at 9 53 11 am" src="https://user-images.githubusercontent.com/26963575/27785428-866f6ac2-5fd5-11e7-90fe-916ff4edb9b2.png">
